src/partials/header-content.hbs

```diff
@@ -38,7 +38,7 @@
  -->
         <div class="navbar-item">
           <span class="control">
-            <a class="button is-primary" href="{{siteRootPath}}/jakarta-ee-docs/{{page.componentVersion.displayVersion}}/jakarta-ee-documentation.pdf">Download</a>
+            <a class="button is-primary" href="{{siteRootPath}}/{{page.component.name}}/{{page.componentVersion.displayVersion}}/jakarta-ee-tutorial.pdf">Download</a>
```

Comment:
Still no clue how to obtain `jakarta-ee-tutorial` via those `{{antora.variables}}`. Closest is `{{page.component.title}}` but it needs to be slugged afterwards.